chrome/content/zotero/xpcom/style.js

```diff
@@ -399,6 +431,72 @@ Zotero.Styles = new function() {
 			}
 		});
 	}
+	
+	/**
+	 * Populate menulist with locales
+	 */
```

Comment:
@dstillman seemed pretty adamant about Zotero.locale fallback, and people might have set all sorts of values on the old "extensions.zotero.export.bibliographyLocale" pref.